### PR TITLE
feat: expose isSystemValue in reference table endpoint(s)

### PR DESCRIPTION
**Pull request by edgarvonk**
Added isSystemValue to RestReferenceTableValue and expose it in reference table endpoint. Added integration test.

Solves PZ-3293

### CODE REVIEW

File: src/itest/kotlin/nl/lifely/zac/itest/ReferenceTableRestServiceTest.kt [hunk 1]
```diff
@@ -95,7 +98,68 @@ class ReferenceTableRestServiceTest : BehaviorSpec({
...
+            val response = itestHttpClient.performGetRequest(
+                "$ZAC_API_URI/referentietabellen/$communicationChannelPageReferenceTableId"
+            )
+            Then(
+                """the provisioned default communicatiekanalen are returned including 'E-formulier'"""
+            ) {
+                val responseBody = response.body!!.string()
+                logger.info { "Response: $responseBody" }
+                response.isSuccessful shouldBe true
+                with(JSONObject(responseBody).toString()) {
```
Comment (hsiliev) [inline src/itest/kotlin/nl/lifely/zac/itest/ReferenceTableRestServiceTest.kt:111]: Do you think that `shouldEqualJson` would simplify this?
The problem would be `id` field that still needs schema validation :(

**Review by hsiliev — APPROVED** [review]

File: src/itest/kotlin/nl/lifely/zac/itest/ReferenceTableRestServiceTest.kt (hunk 1 above)
Comment (edgarvonk) [reply src/itest/kotlin/nl/lifely/zac/itest/ReferenceTableRestServiceTest.kt:111]: yeah probably. I did not spent a lot of time on simplifying this test code to be honest.

Comment (edgarvonk) [reply src/itest/kotlin/nl/lifely/zac/itest/ReferenceTableRestServiceTest.kt:111]: I will update the tests

Comment (edgarvonk) [reply src/itest/kotlin/nl/lifely/zac/itest/ReferenceTableRestServiceTest.kt:113]: did this now. much better indeed :-)

**Pull request merged**